stable/rocketchat/templates/deployment.yaml

```diff
@@ -0,0 +1,65 @@
+{{- if include "host" . -}}
```

Comment:
The linter is also complaining about it and I the pod is crashing with the following when it is not set:

```
/app/bundle/programs/server/boot.js:356
}).run();
   ^

Error: $ROOT_URL, if specified, must be an URL
    at Error (native)
    at packages/meteor.js:1218:13
    at packages/meteor.js:1233:4
    at packages/meteor.js:1380:3
    at /app/bundle/programs/server/boot.js:303:34
    at Array.forEach (native)
    at Function._.each._.forEach (/app/bundle/programs/server/node_modules/underscore/underscore.js:79:11)
    at /app/bundle/programs/server/boot.js:128:5
    at /app/bundle/programs/server/boot.js:352:5
    at Function.run (/app/bundle/programs/server/profile.js:510:12)
```

If this is a required field look at what ghost and gitlab-ce have done for this use case.